state_machine/applications/system-check/main.py

```diff
@@ -29,6 +34,8 @@
     ("Coil Driver Test", "coil", tests.coil_test, True),
     ("Burnwire Test", "burn", tests.burnwire_test, False),
     ("I2C_Scan", "i2c", tests.i2c_scan, False),
+    ("NVM Test", "nvm", tests.nvm_access_test, True),
```

Comment:
Per my comment below about resetting the NVM, I think this should be renamed to "Reset and test NVM"

Reply:
It should be clear here that the test modifies the NVM